modules/osd-monitoring-querying-metrics.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-managing-metrics.adoc
+
+[id="querying-metrics_{context}"]
+= Querying metrics
+
+The OpenShift monitoring dashboard lets you run Prometheus Query Language (PromQL) queries to examine metrics visualized on a plot. This functionality provides information about the state of a cluster and any user-defined projects that you are monitoring.
+
+As a `dedicated-admin`, you can query one namespace at a time for metrics about user-defined projects.
```

Comment:
admin can query more than one namespace at a time, should delete 'one namespace at a time for'

Reply:
Changed it to "one or more"